python-json-rbac/core.py

```
@@ -38,8 +49,20 @@ def _load_key(path: Optional[str], is_private: bool = False) -> str:
+        with open(path, "rb") as f:
+            key_content = f.read().decode()
+        
+        # Basic validation for RSA key format
+        if is_private and "PRIVATE KEY" not in key_content:
+            raise RuntimeError(f"Invalid private key format in {path}")
+        elif not is_private and "PUBLIC KEY" not in key_content:
+            raise RuntimeError(f"Invalid public key format in {path}")
+            
+        return key_content
+    except Exception as e:
+        raise RuntimeError(f"Failed to load key from {path}: {e}")
```

Ruff (0.12.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)